README.md

```diff
@@ -2,6 +2,8 @@
 
 Created due to https://github.com/kubernetes/org/issues/715.
 
+See [The KEP proposal for architecture and details](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1281-network-proxy#proposal).
```

Comment:
grammer nit: _the_ should start with a non-capital _t_.

Reply:
apologies, fixed.